bin/build_release

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+CURRENT_DIR=$(cd $(dirname "$0"); pwd)
+
+# xgo because it allows cross-compilation
+GORELEASER_IMAGE="cyberark/goreleaser:latest-xgo"
+
+echo "Current dir: $CURRENT_DIR"
+
+# TODO: the image cyberark/goreleaser:latest-xgo will need to be pushed
+# TODO: to Dockerhub, and the command below should become
```

Comment:
Probably okay, and I see this done in other build_release files, but would it be clearer to have a single `TODO`, with the subsequent lines indented? I don't have a strong preference either way.

Reply:
Just making sure I understand... would we push this to Dockerhub in the future because this image will be shared by many projects? Otherwise, doesn't seem like much of a burden to build an image each time.

Reply:
Yup the plan is for this to be a standalone image housed somewhere for public consumption by many projects